app/serve.go

```diff
@@ -29,10 +29,14 @@ func (s *ServeService) Run(c *cli.Context) {
 
 	go s.Slacker.BootRTM()
 
+	host := "localhost"
```

Comment:
Defaulting `host` to `""` would be more appropriate since it would cause Phabulous to listen on any address by default.